hnsw/indexHNSW.py

```diff
@@ -0,0 +1,114 @@
+import os
+import helper
+import hnswlib
+import numpy as np
+
+
+class Indexer:
+    def __init__(self, destination_directory):
+        self.path_prefix = destination_directory
+        CORD19_HNSW_INDEX_NAME = 'cord19-hnsw-index'
+        self.folder_path = os.path.join(
+            self.path_prefix, CORD19_HNSW_INDEX_NAME)
+
+    def get_path(self, path) -> str:
+        return os.path.join(self.folder_path, path)
+
+    def download_data(self) -> None:
```

Comment:
I think it's better to keep this function in the bash script instead of in python, that way all the URLs are in one place